### ISSUE TITLE
[v6] AdMob RewardedAd throws error

### ISSUE THREAD

**Issue by pistou** [invertase/react-native-firebase#2584]
## Issue

I'm trying to use AdMob's `RewardedAd` from v6, according to the example code, and it throws an error:

> RNFBAdMobRewardedModule.rewardedShow got 4 arguments, expected 5

![Screenshot_20190915_132920_com kitstrong 1](https://user-images.githubusercontent.com/6703711/64920914-14d7f980-d7bd-11e9-86f2-446a870abdc7.jpg)

**This is only tested on Android. I don't have an iOS setup to try it.**

Sample code:
```JavaScript
import React from 'react';
import {
    RewardedAdEventType,
    RewardedAd,
    TestIds,
} from '@react-native-firebase/admob';
import { View } from 'react-native';

const Home = () => {
    const rewarded = RewardedAd.createForAdRequest(TestIds.REWARDED, {
        requestNonPersonalizedAdsOnly: true,
    });

    rewarded.onAdEvent((type, error, reward) => {
        if (type === RewardedAdEventType.LOADED) {
            rewarded.show();
        }
        if (type === RewardedAdEventType.EARNED_REWARD) {
            console.log('User earned reward of ', reward);
        }
    });
    rewarded.load();

    return <View />;
};
export default Home;
```

---

### iOS

<details><summary>Click To Expand</summary>
<p>

#### `ios/Podfile`:

- [x] I'm not using Pods
- [ ] I'm using Pods and my Podfile looks like:

```ruby
# N/A
```

#### `AppDelegate.m`:

```objc
// N/A
```

</p>
</details>

---

### Android

<details><summary>Click To Expand</summary>
<p>

#### Have you converted to AndroidX?

- [ ] my application is an AndroidX application?
- [ ] I am using `android/gradle.settings` `jetifier=true` for Android compatibility?
- [ ] I am using the NPM package `jetifier` for react-native compatibility?

#### `android/build.gradle`:

```groovy
// N/A
```

#### `android/app/build.gradle`:

```groovy
// N/A
```

#### `android/settings.gradle`:

```groovy
// N/A
```

#### `MainApplication.java`:

```java
// N/A
```

#### `AndroidManifest.xml`:

```xml
<!-- N/A -->
```

</p>
</details>

---

## Environment

<details><summary>Click To Expand</summary>
<p>

**`react-native info` output:**

```
System:
    OS: Windows 10
    CPU: (8) x64 Intel(R) Core(TM) i7-2600K CPU @ 3.40GHz
    Memory: 10.69 GB / 15.90 GB
  Binaries:
    Node: 10.15.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.17.3 - D:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.11.3 - C:\Program Files\nodejs\npm.CMD
```

- **Platform that you're experiencing the issue on**:
  - [ ] iOS
  - [x] Android
  - [ ] **iOS** but have not tested behavior on Android
  - [x] **Android** but have not tested behavior on iOS
  - [ ] Both
- **`react-native-firebase` version you're using that has this issue:**
  - `react@16.9.0`
  - `react-native@0.60.5`
  - `@react-native-firebase/app@0.4.0`
  - `@react-native-firebase/admob@0.4.0`
- **Are you using `TypeScript`?**
  - `N`
  
</p>
</details>

---

Think `react-native-firebase` is great? Please consider supporting all of the project maintainers and contributors by donating via our [Open Collective](https://opencollective.com/react-native-firebase/donate) where all contributors can submit expenses. [[Learn More]](https://invertase.io/oss/react-native-firebase/contributing/donations-expenses)

- 👉 Check out [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter for updates on the library.

**Comment by Salakar** [comment]
cc @ehesp - looks like the wrong number of arguments are being sent to the native method

**Comment by Ehesp** [comment]
Woops - I see the problem. Will send up a fix.

**Comment by Ehesp** [comment]
Thanks for the report. This will be in the next release!

**Comment by Salakar** [comment]
@pistou thanks for the report and also the PR you sent up, PR & fix for this are live now in v0.4.2 thanks
